apps/nestjs-api/src/reminder-emails/reminder-emails.controller.ts

```diff
@@ -0,0 +1,116 @@
+import { Controller, Get, UseGuards } from '@nestjs/common'
+import { UserType } from '@talent-connect/common-types'
+import { CronjobAuthGuard } from '../auth/cronjob-auth.guard'
+import { ReminderEmailsService } from './reminder-emails.service'
+
+@Controller('reminder-emails')
+@UseGuards(CronjobAuthGuard)
+export class ReminderEmailsController {
+  constructor(private readonly reminderEmailsService: ReminderEmailsService) {}
+
+  @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/mentors-complete-profile')
+  async sendMentorCompleteProfileReminders() {
+    const mentorsWithDraftingProfile =
+      await this.reminderEmailsService.getDraftingConProfiles({
+        userType: UserType.MENTOR,
+      })
+
+    if (mentorsWithDraftingProfile.length > 0) {
+      // send reminder emails
+      mentorsWithDraftingProfile.forEach(async (mentor) => {
+        await this.reminderEmailsService.sendCompleteProfileReminder({
+          userType: UserType.MENTOR,
+          email: mentor.props.email,
+          firstName: mentor.props.firstName,
+        })
+      })
+    }
+
+    return {
+      message: `Complete Profile reminder emails sent to ${mentorsWithDraftingProfile.length} mentors`,
+    }
+  }
+
+  @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/mentees-complete-profile')
+  async sendMenteeCompleteProfileReminders() {
+    const menteesWithDrafingProfile =
+      await this.reminderEmailsService.getDraftingConProfiles({
+        userType: UserType.MENTEE,
+      })
+
```

Comment:
Same here. We want to send this reminder to mentee profiles that were created 14 days ago AND have Drafting profile status. Do we also filter out the drafting mentee profiles by age?